modules/internal-load-balancer/variables.tf

```
@@ -1,32 +1,43 @@
+variable "disk_image" {
+  default     = "ubuntu-1804-lts"
+  description = "The image from which the main compute instance disks will be initialized. The supported images are: ubuntu-1604-lts; ubuntu-1804-lts; rhel-7."
+  type        = string
+}
+
+variable "disk_size" {
+  default     = 40
+  description = "The size of var.disk_image, expressed in units of gigabytes."
+  type        = number
+}
 
 variable "labels" {
   default     = {}
-  description = "A collection of labels which will be applied to resources."
+  description = "The labels which will be applied to the compute instances."
   type        = map(string)
 }
 
-variable "prefix" {
-  description = "The prefix which will be prepended to the names of resources."
+variable "machine_type" {
+  default     = "n1-standard-8"
+  description = "The identifier of the set of virtualized hardware resources which will be available to the compute instances."
```

do we want to add a link to the description here to link to options? This reads very well aligned to Google's documentation tone, however the "identifier of the set of ...." caught me off guard and I had to reread it to process it correctly.

a la 

```suggestion
  description = "The identifier of the set of virtualized hardware resources which will be available to the compute instances. More details on machine types can be found at https://cloud.google.com/compute/docs/machine-types"